pulumi/Pulumi.dev.yaml

```diff
@@ -0,0 +1,6 @@
+config:
+  aws:region: us-east-1
```

Comment:
Can you please switch to `eu-west-2`. This is our default region.

Reply:
Now it's changed :)